deeppixel/image_stitching_2/code/pano.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import cv2
+import sys
+from matchers import matchers
+
+class Stitch:
+	def __init__(self, args):
+		self.path = args
+		fp = open(self.path, 'r')
```

Comment:
This is fine. But for better practices, i would suggest look up stackoverflow or python docs on how to read all the files from a given folder and take the relative path of that folder as an input